packages/swap/src/translators/constants.ts

```diff
@@ -0,0 +1,10 @@
+import {Swap} from '@yoroi/types'
+
+export const supported_protocols: ReadonlyArray<Swap.SupportedProtocol> = [
+  'minswap',
+  'wingriders',
+  'sundaeswap',
+  'muesliswap',
+  'muesliswap_v2',
+  'vyfi',
```

Comment:
are we supporting `vyfi` I thought it was using different address logic, and it was throwing some errors?

Reply:
hmm, I thought we supported it as is officially supported by museliswap as well

Reply:
supportedProviders (cuz protocol is used only in the create order) everywhere else they are using Providers.